components/ui-api-layer/internal/domain/application/app_service.go

```diff
@@ -290,8 +293,14 @@ func (svc *applicationService) extractErrorCause(body io.ReadCloser) error {
 }
 
 func (svc *applicationService) drainAndCloseBody(body io.ReadCloser) {
-	_ = iosafety.DrainReader(body)
-	body.Close()
+	err := iosafety.DrainReader(body)
+	if err != nil {
+		glog.Warningf("Unable to drain body reader. Cause: %v", err)
```

Comment:
maybe it would be good to return these errors. If this is not an option, log them with error level